packages/components/src/Notes.js

```diff
@@ -1,15 +1,31 @@
-import React, { useEffect } from 'react'
+import React, { useEffect, useRef } from 'react'
 import { useDeck } from './context'
+import { useStep } from './useStep'
 
 export const Notes = props => {
   const context = useDeck()
+  if (!context || !context.register) {
+    // The slide was rendered as a preview by the <Presenter> component.
+    return false
```

Comment:
We can break the rule of "no conditional hooks" here, because slides rendered for the `<Presenter>` preview will never be re-render *with* context. They always have **no** context.